index.d.ts

```diff
@@ -0,0 +1,33 @@
+import { CSSProperties, FC, ReactNode } from "react";
```

Comment:
I would prefer to move this file to `/types/index.d.ts` and change `package.json` accordingly.